src/app/(with-header)/activities/[id]/components/Title.tsx

```
@@ -29,10 +30,10 @@
         </div>
       </div>
 
-      {isDropDown && (
-        <div className='mt-30 flex items-center gap-1'>
+      {isOwner && (
+        <button>
           <IconDropdown />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)